src/components/GoodsList/GoodsList.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const GoodList = (props) => {
+  const {
+    goods, minLength, handleReverseList,
```

Comment:
переноси каждое свойство
```suggestion
    goods,
    minLength,
    handleReverseList,
```